package.json

```diff
@@ -95,6 +96,7 @@
 		"unzip-stream": "^0.3.4",
 		"validator": "^13.12.0",
 		"vscode-kubernetes-tools-api": "^1.3.0",
+		"vscode-openshift-connector": "file:",
```

Comment:
@msivasubramaniaan Something strange happens with this `diff` functionality.... When pulled locally from your PR I really cannot fine this line 99 added:
```
		"vscode-openshift-connector": "file:",
```

whle GH still shows me this:

![image](https://github.com/user-attachments/assets/916bd9a4-312f-49e8-a31f-89d7a7002c94)

Also, the PR cannot be auto-rebased on top of the main branch, so...

Could you please squash your commits and rebase on top of the current master branch so it be easier to browse/search through the change?

Reply:
Also, this PR quite hardly conflicts with previously added alizer and related integration test

Reply:
> Also, the PR cannot be auto-rebased on top of the main branch, so...

I think that, since there are merge commits in this PR, we will need to use squash and merge instead of rebase and merge, since we should avoid having merge commits in the repo.